compiler/env/OMRVMEnv.cpp

```diff
@@ -118,6 +118,12 @@ OMR::VMEnv::getHighResClockResolution(OMR_VMThread *omrVMThread)
    return ::highResClockResolution();
    }
 
+uint64_t 
+OMR::VMEnv::getHighResClockResolution(TR_FrontEnd *fe)
```

Comment:
Do we need to add a front-end parameter here at all? Shouldn't there just be a getHighResClockResolution() call (no parameter) and everything should use that?

Would you entertain the notion to add this call without a parameter and modify your calls to use that, then do another quick PR to change the remaining calls to getHighResClockResolution() that pass alternate parameters? (I only see 6 other calls to the "comp()" one and none to the OMRVMThread one).

Or you could just add a second commit to clean those up in this PR...